mlir/include/mlir/Dialect/Arith/IR/Arith.h

```diff
@@ -65,6 +67,10 @@ class ConstantIntOp : public arith::ConstantOp {
   static void build(OpBuilder &builder, OperationState &result, int64_t value,
                     Type type);
 
+  /// Build a constant int op that produces an integer from an APInt
+  static void build(OpBuilder &builder, OperationState &result, Type type,
+                    const APInt &value);
```

Comment:
Changing the order between type and value in this constructor with respect to other constructors is highly confusing. Please revise.

Reply:
https://llvm.org/docs/GitHub.html#stacked-pull-requests